recipes-tools/fluent-bit/files/fluent-bit.init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 ### BEGIN INIT INFO
 # Provides:          fluent-bit
-# Required-Start:    $local_fs $network $named $time $syslog
+# Required-Start:    $local_fs $network $named $time $syslog fetch-config
```

Comment:
I thought this was not honored, right? unless just for readability purposes

Reply:
yes, I tested and it is not honored. I'd like to leave it there anyways in case we find a solution to make it work.